java-client/src/main/java/com/cumulocity/sdk/client/RestConnector.java

```diff
@@ -114,8 +114,13 @@ public boolean isClosed() {
 
     @Override
     public <T extends ResourceRepresentation> T get(String path, CumulocityMediaType mediaType, Class<T> responseType) throws SDKException {
-        Response response = getClientResponse(path, mediaType);
-        return responseParser.parse(response, responseType, OK.getStatusCode());
+        try {
+            Response response = getClientResponse(path, mediaType);
+            return responseParser.parse(response, responseType, OK.getStatusCode());
+        } catch (SDKException e) {
+              log.error(" request : GET {} for tenant: {} user: {}", path, platformParameters.getTenantId(), platformParameters.getUser());
```

Comment:
I might be wrong but for me it looks like that a `SDKException` is only thrown when parsing the response in all scenarios. I would therefore purpose to also include the response body in the log message.